modular_skyrat/master_files/code/modules/client/preferences/tgui_prefs_migration.dm

```diff
@@ -20,6 +20,7 @@ MUTANT_SYNTH_CHASSIS, \
 "taur", \
 "xenodorsal", \
 "xenohead", \
+"pod_hair", \
```

Comment:
Are we sure that it's going to do it properly in this case? Do we perhaps have to increment the savefile version if this needs to be migrated?